string robot_info
---
bool success